capitalize_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './decorator'
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] `upcase` method can convert the given string to all block letters ie: `STRING`. You do need to use the `capitalize` method which will only convert the first letter of the given string ie: `String`. Please make this change.